arcade/text.py

```diff
@@ -446,28 +446,28 @@ def left(self) -> int:
         """
         Pixel location of the left content border.
         """
-        return self._label._get_left()
+        return self._label._get_left_anchor()
```

Comment:
I don't see this method in current use in label yet?

Reply:
Removed per the changes caffienepills suggested.